index.html

```diff
@@ -5612,25 +5533,174 @@ <h3>Remote Document and Context Retrieval</h3>
+              of <a data-link-for="LoadDocumentCallback">url</a>, as defined in [[HTML]],
+              using the existing <var>documentUrl</var> as the document's URL.
+              <div class="issue atrisk">
+                The use of the <a data-cite="HTML/urls-and-fetching.html#document-base-url">Document Base URL</a>
+                from [[HTML]] for setting the <a>base IRI</a> of the enclosed JSON-LD
+                is an experimental feature, which may be changed in a future version of this specification.
+              </div>
+            </li>
+            <li>If the <a data-link-for="LoadDocumentCallback">url</a> parameter
+              contains a <a data-cite="RFC3986#section-3.5">fragment identifier</a>,
+              set <var>source</var> to the <a data-cite="DOM#dom-node-textcontent">textContent</a>
+              of the <a data-cite="HTML/scripting.html#the-script-element">script element</a> in <var>document</var>
+              having an <a data-cite="HTML/dom.html#the-id-attribute">id attribute</a>
+              that matches the fragment identifier, after decoding <a data-cite="RFC3986#section-2.1">percent encoded sequences</a>.
+              <p>If no such element is found,
+                or the located element is not a <a>JSON-LD script element</a>,
+                the <var>promise</var> is rejected with a <a>JsonLdError</a> whose code is set to <a data-link-for="JsonLdErrorCode">loading document failed</a>
+                and processing is terminated.</p>
+            </li>
+            <li>Otherwise, if the <a data-link-for="LoadDocumentOptions">profile</a>
+              option is specified,
+              set <var>source</var> to the result of transforming the
+              <a data-cite="DOM#dom-node-textcontent">textContent</a>
+              of the first <a data-cite="HTML/scripting.html#the-script-element">script element</a> in <var>document</var>
+              having an <a data-cite="HTML/semantics.html#attr-link-type">type attribute</a>
+              of <code>application/ld+json</code> along with the value of the
+              <a data-link-for="LoadDocumentOptions">profile</a> option, if found.</li>
+            <li>If <var>source</var> is undefined and the <a data-link-for="LoadDocumentOptions">extractAllScripts</a> option is not present, or <code>false</code>,
+              set <var>source</var> to the <a data-cite="DOM#dom-node-textcontent">textContent</a>
+              of the first <a>JSON-LD script element</a> in <var>document</var>.
+              <p>If no such element is found,
+                or the located element is not a <a>JSON-LD script element</a>,
+                the <var>promise</var> is rejected with a <a>JsonLdError</a> whose code is set to <a data-link-for="JsonLdErrorCode">loading document failed</a>
+                and processing is terminated.</p></li>
+            <li>If <var>source</var> is defined and the,
```

Comment:
Something is missing between "and the" and the trailing comma.

Reply:
This should simply remove the "and the".